### PR TITLE
EPAM, NodeJS practice, homework2

### PR DESCRIPTION
**Pull request by grechintanya**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import express, { NextFunction, Request, Response } from 'express';
+import path from 'path';
+import healthCheckRouter from './routes/healthCheck';
+
+const app = express();
+const port: Number = 3000;
+
+import swaggerDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
```
Comment (IhorOsadchyi) [inline src/server.ts]: Better to move these import up, there all other imports located
```suggestion

```

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import express, { NextFunction, Request, Response } from 'express';
+import path from 'path';
+import healthCheckRouter from './routes/healthCheck';
```
Comment (IhorOsadchyi) [inline src/server.ts]:
```suggestion
import express, { NextFunction, Request, Response } from 'express';
import swaggerDoc from 'swagger-jsdoc';
import swaggerUi from 'swagger-ui-express';
import path from 'path';
import healthCheckRouter from './routes/healthCheck';
```

**Review by IhorOsadchyi — COMMENTED** [review]
Please, add `.gitignore` file and place at least `node_modules` and `build` folders into it for exclusion.

File: src/server.ts (hunk 2 above)
Comment (grechintanya) [reply src/server.ts]: Hi, I've made corrections

**Review by IhorOsadchyi — APPROVED** [review]

**Pull request merged**